blockchain/block.go

```diff
@@ -0,0 +1,53 @@
+package blockchain
+
+import (
+	"crypto/sha256"
+	"encoding/hex"
+	"fmt"
+	"strconv"
+	"time"
+)
+
+type Block struct {
+	Index        int
+	Timestamp    int64
+	Content      string
+	PreviousHash string
+	Hash         string
```

Comment:
i'd say that we can keep hash as []byte. Go encodes those as base64 strings if we ever need a text representation anyway, i think it will make certain operations easier in the future